m_etcd/taskmgr.go

```diff
@@ -86,22 +86,22 @@ func (m *taskManager) add(taskID string) bool {
 	if err != nil {
 		etcdErr, ok := err.(*etcd.EtcdError)
 		if !ok || etcdErr.ErrorCode != EcodeNodeExist {
-			m.ctx.Log(metafora.LogLevelError, "Claim of %s failed with an unexpected error: %v", key, err)
+			metafora.Errorf("Claim of %s failed with an unexpected error: %v", key, err)
 		} else {
-			m.ctx.Log(metafora.LogLevelInfo, "Claim of %s failed, already claimed", key)
+			metafora.Debugf("Claim of %s failed, already claimed", key)
```

Comment:
Ha, yeah. It confused *everybody.* Totally looked like something bad was happening.